AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt

```diff
@@ -13,6 +13,9 @@
 ||geolocation.onetrust.com/cookieconsentpub/*/geo/location$xmlhttprequest,redirect-rule=noopjson
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/184091
+cooppank.ee#@##cookie-modal
```

Comment:
The cookies should also be blocked, but like @piquark6046 said. You can try using `#$#` rules to do this.

Reply:
@piquark6046 I see the horizontal scrolling at the bottom after adding the css rules and an issue with the video, so `trusted-set-cookie` scriptlet seems to be safer here.

Reply:
I'd just add this to specific.txt.
```adblock
cooppank.ee#%#//scriptlet('trusted-set-cookie', 'cookie-consent', 'necessary%2B1721979799357')
```

Reply:
@alexx7311 please add the exceptions back)